### ISSUE TITLE
Symfony: Missing translation key, with given domain

### ISSUE THREAD

**Issue by ingfdoaguirre** [Haehnchen/idea-php-symfony2-plugin#1874]
Hi, I'm having a problem with an error on missing translation key when I specify the domain in trans filter.

`{{ 'loading_table', 'loading.table.sales'|trans([], 'messages') }}`

The key exists in my yaml file (domain)

Im using the latest plugin version 2022.1.222 and Im a premium user.

Thank you!

**Comment by jeffhinshaw** [comment]
I have the same issue, but I noticed using an empty hash `{}` instead of an empty array `[]` for the translation parameters fixed the issue for me.

```twig
{{ 'loading.table.sales'|trans({}, 'messages') }}
```

**Comment by gndk** [comment]
Same issue here on 2022.1.230 and premium.

The fix from jeffhinshaw works for me, too. However it would be nice if the default syntax would be supported again.

I can't remember when I first noticed this, but it was working fine until recently.

**Comment by Haehnchen** [comment]
So finally all infrastructure issues are resolved on PhpStorm 2022.2; tests are running again inside 2022.x builds, which are also notifying the issue:

```
TwigTranslationCompletionContributorTest > testTwigTransCompletion FAILED
    junit.framework.AssertionFailedError at TwigTranslationCompletionContributorTest.java:24
```

will be fixed via: https://github.com/Haehnchen/idea-php-symfony2-plugin/pull/1827

**Comment by Haehnchen** [comment]
fixed
